caproto/asyncio/client.py

```diff
@@ -135,6 +135,12 @@ async def disconnect(self):
         await self._tasks.cancel_all(wait=True)
         self.log.debug('Broadcaster: Closing the UDP socket')
         self.udp_sock = None
+        try:
+            if self.protocol is not None:
+                if self.protocol.transport is not None:
+                    self.protocol.transport.close()
+        except OSError:
+            self.log.debug('Broadcaster transport close error')
```

Comment:
If we make these `exception` instead of `debug` it will be easier to get detailed bug reports.

Reply:
```suggestion
            self.log.exception('Broadcaster transport close error')
```